.github/workflows/_reusable_run_tests.yml

```diff
@@ -123,6 +123,13 @@ jobs:
           path: Carthage
           key: ${{ runner.os }}-xcode${{ steps.xcode-version.outputs.XCODE_VERSION }}-carthage-${{ hashFiles('Cartfile.resolved') }}
 
+      - name: Restore Danger results
+        id: restore-danger-results
+        uses: actions/cache/restore@v4
+        with:
+          path: Danger.results
+          key: ${{ runner.os }}-Danger.results
```

Comment:
Maybe the Xcode version should be included in the cache key, as new versions might have more or less warnings?
`xcode${{ steps.xcode-version.outputs.XCODE_VERSION }}`

Reply:
done, thanks